.travis.yml

```diff
@@ -6,11 +6,15 @@ jobs:
     # macOS
     - env: TARGET_MACHINE=i3osx
       os: osx
-    - env: TARGET_MACHINE=ti3osx
+    - env:
+      - TARGET_MACHINE=ti3osx
+      - PARALLEL_MATS=1
       os: osx
     - env: TARGET_MACHINE=a6osx
       os: osx
-    - env: TARGET_MACHINE=ta6osx
+    - env:
+      - TARGET_MACHINE=ta6osx
+      - PARALLEL_MATS=1
```

Comment:
So these only run well the test is not run in parallel due to test machine limitations?